src/ResourceManagement/Insights/Microsoft.Azure.Insights/Generated/Insights/MetricDefinitionsOperations.cs

```diff
@@ -96,7 +97,6 @@ internal MetricDefinitionsOperations(InsightsClient client)
             var _baseUrl = this.Client.BaseUri.AbsoluteUri;
             var _url = new System.Uri(new System.Uri(_baseUrl + (_baseUrl.EndsWith("/") ? "" : "/")), "{resourceUri}/providers/microsoft.insights/metricDefinitions").ToString();
             _url = _url.Replace("{resourceUri}", resourceUri);
-            _url = _url.Replace("{subscriptionId}", System.Uri.EscapeDataString(this.Client.SubscriptionId));
```

Comment:
I any case the metrics / metrics definitions APIs were marked in the last version of PowerShell as "to deprecated and replaced".